R/upkeep.R

```diff
@@ -118,86 +122,56 @@ use_tidy_upkeep_issue <- function(year = NULL) {
 
 tidy_upkeep_checklist <- function(year = NULL,
                                   posit_pkg = is_posit_pkg(),
-                                  posit_person_ok = is_posit_person_canonical()) {
+                                  posit_person_ok = is_posit_person_canonical(),
+                                  repo_spec = "OWNER/REPO") {
   year <- year %||% 2000
 
   bullets <- c()
 
   if (year <= 2000) {
     bullets <- c(
       bullets,
-      "Pre-history",
+      "### Pre-history",
       "",
       todo("`usethis::use_readme_rmd()`"),
       todo("`usethis::use_roxygen_md()`"),
       todo("`usethis::use_github_links()`"),
       todo("`usethis::use_pkgdown_github_pages()`"),
       todo("`usethis::use_tidy_github_labels()`"),
       todo("`usethis::use_tidy_style()`"),
-      todo("`usethis::use_tidy_description()`"),
       todo("`urlchecker::url_check()`"),
       ""
     )
   }
   if (year <= 2020) {
     bullets <- c(
       bullets,
-      "2020",
+      "### 2020",
       "",
-      todo("
-        `usethis::use_package_doc()`
-        Consider letting usethis manage your `@importFrom` directives here.
-        `usethis::use_import_from()` is handy for this."),
-      todo("
-        `usethis::use_testthat(3)` and upgrade to 3e, \\
-        [testthat 3e vignette](https://testthat.r-lib.org/articles/third-edition.html)"),
-      todo("
-        Align the names of `R/` files and `test/` files for workflow happiness.
-        The docs for `usethis::use_r()` include a helpful script.
-        `usethis::rename_files()` may be be useful."),
+      todo("`usethis::use_package_doc()`"),
+      todo("`usethis::use_testthat(3)`"),
+      todo("Align the names of `R/` files and `test/` files"),
```

Comment:
In general, I like having detail, links, etc. But I can also imagine this is about making the items for the distant past more lean, yeah? And saving detail for the items we are more likely to be engaging with in the year of our lord 2023?

Reply:
Yeah, exactly.